routing-updated/src/agent.config.js

```diff
@@ -1,28 +1,25 @@
-const { Finding, FindingSeverity, FindingType } = require('forta-agent');
+const { Finding, FindingSeverity, FindingType } = require("forta-agent");
```

Comment:
The bot uses eslint with the airbnb rules and it expects all strings to use single quotes